test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java

```diff
@@ -88,7 +89,7 @@ private void cleanup(AccumuloClient c, String[] tableNames) throws Exception {
   }
 
   private void createTable(AccumuloClient c, String tableName) throws Exception {
-    c.tableOperations().create(tableName);
+    c.tableOperations().create(tableName, new NewTableConfiguration());
```

Comment:
Why add the empty `new NewTableConfiguration()`?

Reply:
I believe that was the intent of this ticket.  The NewTableConfiguration does some sort of checking and validating and down the road maybe more default behavior.  BUT I'm game for whatever is desired.

Reply:
`tableOperations().create(tableName)` and `c.tableOperations().create(tableName, new NewTableConfiguration())` should be functionally equivalent.  I suspect the intent of the ticket was to add `NewTableConfiguration` when there were things to set on it.